app/models/recruitment_cycle.rb

```diff
@@ -15,6 +15,11 @@ def current_recruitment_cycle
     end
     alias_method :current, :current_recruitment_cycle
 
+    def current_allocation_recruitment_cycle
```

Comment:
I don't think `RecruitmentCycle` should know about allocations. This should be on `Allocation` (or somewhere else)

Reply:
Is this only being used in the specs too? If so it should probably just be defined in the factory rather than out in the code.

Reply:
Sure, removed as not being used 👍